packages/node-collector/src/NodeCollector/ParsedNodeCollector.php

```diff
@@ -210,12 +210,12 @@ public function findNewsByClass(string $className): array
     {
         $newsByClass = [];
 
-        foreach ($this->news as $new) {
-            if (! $this->nodeNameResolver->isName($new->class, $className)) {
+        foreach ($this->news as $singleNews) {
+            if (! $this->nodeNameResolver->isName($singleNews->class, $className)) {
```

Comment:
This should be kept, as it's not a typical news as in '"today's news"

Reply:
any service other than Inflector that can handle it?

Reply:
implemented.